src/js/orders.spec.ts

```diff
@@ -4,7 +4,7 @@ const abi = require('ethereumjs-abi');
 import {utils, Wallet} from 'ethers';
```

Comment:
Not sure what this is all about (line 3)

![Screenshot 2020-09-15 at 9 33 12 AM](https://user-images.githubusercontent.com/11778116/93179830-859a5480-f736-11ea-8547-f5ecc3eae3be.png)